src/compile/layer.ts

```diff
@@ -113,6 +113,7 @@ export class LayerModel extends Model {
       child.parseScale();
 
       // FIXME: correctly implement independent scale
+      // also need to check whether the scales are actually compatible, e.g. use the same sort
```

Comment:
- Missing test for this case.
- Why don't we just check it now?  It shouldn't be that hard?

If you really prefer not to, then please change the comment to:
```
// FIXME(#1602): check whether the scales are actually compatible, e.g. use the same sort
```